android/main/java/com/couchbase/lite/internal/CouchbaseLiteInternal.java

```diff
@@ -168,6 +169,10 @@ public static Map<String, String> loadErrorMessages(@NonNull Context ctxt) {
         return errorMessages;
     }
 
+    @VisibleForTesting
+    public static void setExtensionPath(@NonNull Context ctxt, @NonNull File extDir, @NonNull File scratchDir) {
```

Comment:
What is the scratchDir for here?

Reply:
It is where the asset used to be copied.

Reply:
Removed